src/integration-tests/bash/cleanup.sh

```diff
@@ -524,31 +524,34 @@ if [ -x "$(command -v helm)" ]; then
   echo "Detected Helm Version [$(helm version --short --client)]"
   echo @@ `timestamp` Deleting installed helm charts
   namespaces=`kubectl get ns | grep -v NAME | awk '{ print $1 }'`
-  for ns in $namespaces
-  do 
-   if [ ! "$DRY_RUN" = "true" ]; then
-     (
-     set -x
-     helm list --short --namespace $ns | while read helm_name; do
-       if [ "$HELM_VERSION" == "V2" ]; then
-         helm delete --purge  $helm_name
-       else 
-         helm uninstall $helm_name -n $ns 
-       fi
-     done
-     )
-   else
-     (
-     helm list --short --namespace $ns | while read helm_name; do
-       if [ "$HELM_VERSION" == "V2" ]; then
-         echo @@ `timestamp` Info: DRYRUN: helm delete --purge  $helm_name
-       else 
-         echo @@ `timestamp` Info: DRYRUN: helm uninstall $helm_name -n $ns 
-       fi
-     done
-     )
-   fi
-  done
+
+  if [ "$HELM_VERSION" == "V2" ]; then
+    helm list --short | while read helm_name; do
+      if [ ! "$DRY_RUN" = "true" ]; then
+        (
+        set -x
+        helm delete --purge  $helm_name
+        )
+      else
+        echo @@ `timestamp` Info: DRYRUN: helm delete --purge  $helm_name
+      fi
+    done
+  else
+    namespaces=`kubectl get ns | grep -v NAME | awk '{ print $1 }'`
+    for ns in $namespaces
```

Comment:
still don't need this loop....
`helm list --all-namespaces`

Reply:
yes, I haven't fixed helm3 ..may be @anpanigr or I can look into helm3. I have only fixed helm2 for now, external jenkins is using helm2. Have something to work on mii, will get back to helm3 later..

Reply:
Helm 3 need an explicit namespace to delete helm chart. So we need to loop thru the same space.  If we can find the list of name spaces dynamically created for the testing, we can avoid looping for all namesapces

Reply:
@anpanigr how about do `helm list --all-namespaces` and get the ns and use that for deletion instead of looping through all the existing namespaces..

Reply:
Why is  this such a big worry? The script takes several seconds to run, and this would only shave off milliseconds.

Reply:
(Just curious if I'm misunderstanding.)

Reply:
yesterday i watched it take over 15 minutes as it ran helm list 483 times

Reply:
sure it will run in a few seconds if there is nothing to delete!

Reply:
more namespaces more time ..

Reply:
You should be able to get the namespace from the table.  If that is hard to parse, `helm list` supports output formats of json and yaml.

Reply:
Oh. That's an impressive namespaces count!

Reply:
yes. helm3 chart deletion needs to be reviewed and fixed. In any case, this PR is just for helm2 fix for now. I have submitted a job on external jenkins with my branch. @markxnelson @rjeberhard you can merge this PR if you are comfortable after seeing the jenkins results.

Reply:
Why are we spending time on fixing helm2 when we dropped helm2 support for 3.0.0?  For this reason, I'm only interested in fixing helm3 and having our external Jenkins use helm3.

Reply:
@anpanigr changed the integration tests for helm3, but the changes were not tested on external jenkins using helm3. That needs to be done along with the helm3 fix in cleanup.sh. I can close this PR if we don't need helm2 fix.

Reply:
ok - well let's make sure the test are in fact only using helm 3 then please :)